openwisp_notifications/tests/test_utils.py

```diff
@@ -105,12 +105,3 @@ def run_check():
                 )
                 error = run_check()
                 self.assertIn(error_message, error.hint)
-
-        with self.subTest('Test setting dotted path is not subclass of ModelAdmin'):
-            path = 'openwisp_notifications.admin.NotificationSettingInline'
-            with patch.object(app_settings, 'IGNORE_ENABLED_ADMIN', [path]):
-                error_message = (
-                    f'"{path}" does not subclasses "django.contrib.admin.ModelAdmin"'
-                )
-                error = run_check()
-                self.assertIn(error_message, error.hint)
```

Comment:
Instead of removing this test, change the path to some other inline class in OpenWISP

`openwisp_users.admin.OrganizationUserInline`